### ISSUE TITLE
Why is the weight of moco_loss 0?

### ISSUE THREAD

**Issue by Monalissaa** [jiamings/d2c#5]
Thanks for your work!
I noticed that on line 488 in train/train.py,
`loss += 0.0*moco_loss`
Why is the weight of moco_loss 0? Are the weights here not changed to the correct values?

**Comment by Monalissaa** [comment]
😔

**Comment by jiamings** [comment]
I believe this is a typo created when we ported the training code (which was cleaned up significantly, and we had included the case without moco_loss in the rebuttal), but I will leave @a7b23 to confirm it. I also recall that the moco_loss has a scale much smaller than the reconstruction loss, so we used a much larger weight for it.

**Comment by a7b23** [comment]
Yes, this was indeed a typo, have fixed it now. Thanks, for pointing this out! As @jiamings pointed out, this was created during the ablations we were running. 
Note that reconstruction_loss is divided by args.recon_loss_weight, so this ensures that moco_loss weight is higher than that of reconstruction loss
